cf/service_broker_test.go

```diff
@@ -209,6 +216,21 @@ var _ = Describe("Client ServiceBroker", func() {
 				Expect(err).ShouldNot(HaveOccurred())
 				assertBrokersFoundMatchTestBroker(1, broker)
 			})
+
+			Context("when the broker is space-scoped", func() {
+				BeforeEach(func() {
+					ccResponse = ccBrokersResponse(ccSpaceScopedBroker)
+					ccResponseCode = http.StatusOK
+				})
```

Comment:
`BeforeEach` does not make sense if you have a single `It`, just move the code at the start of the `It`